server/test/services/program/ProgramServiceTest.java

```diff
@@ -769,7 +769,8 @@ public void updateProgram_withNoProgram_throwsProgramNotFoundException() {
                     /* eligibilityIsGating= */ true,
                     ProgramType.DEFAULT,
                     /* isIntakeFormFeatureEnabled= */ false,
-                    ImmutableList.copyOf(new ArrayList<>())))
+                    ImmutableList.copyOf(new ArrayList<>()),
```

Comment:
This was what was already there.  It's for the `tiGroups` parameter.  I just had to add a comma so that I could add the following line with the categories.